pkg/imagestreamregister/imagestream_register.go

```diff
@@ -0,0 +1,52 @@
+package imagestreamregister
```

Comment:
The only reason this is included is because I needed to change the file. The line below is removed, otherwise this is identical to the upstream file. Without this problematic line upstream, I'd just reference the remote/vendor code like we're doing with openshift apps.
https://github.com/openshift/api/blob/master/image/v1/register.go#L48

Reply:
That's right. I remember our conversation on this now. Thoughts on moving to `pkg/apis/addtoscheme_imagev1.go`? @jortel do you have any input on a good place to put this?

Reply:
Anywhere works for me, as long as it's a separate package. With all of the package-exported variables there, I'd be afraid that putting it in an existing package could clash (or potentially clash in the future).